src/frontend/Komponenter/Behandling/Brev/ManueltBrev.tsx

```diff
@@ -34,15 +32,19 @@ const BrevKolonner = styled.div`
     display: flex;
 `;
 
-const BrevWrapper = styled.div`
-    margin-left: 4rem;
-`;
-
-interface Props {
-    fagsakId: string;
-}
-
-const ManueltBrev: React.FC<Props> = ({ fagsakId }) => {
+type Props =
+    | {
+          oppdaterBrevressurs: (brevRessurs: Ressurs<string>) => void;
+          fagsakId: string;
+          behandlingId?: string;
+      }
+    | {
+          oppdaterBrevressurs: (brevRessurs: Ressurs<string>) => void;
+          fagsakId?: string;
+          behandlingId: string;
+      };
```

Comment:
Noen som vet om en penere måte å gjøre dette? Vi vil at man må ha med enten `fagsakId` eller `behandlingId`

Reply:
Nei, men du vil vel isåfall ha
```
type Props =
    | {
          oppdaterBrevressurs: (brevRessurs: Ressurs<string>) => void;
          fagsakId: string;
      }
    | {
          oppdaterBrevressurs: (brevRessurs: Ressurs<string>) => void;
          behandlingId: string;
      };
```

eller?

Reply:
Ja, egentlig.

Reply:
Fikset i 2690ebf